tests/testthat/test-set_status.R

```diff
@@ -30,7 +31,7 @@ test_that("`set_status`: Basic expectations", {
 
   # expect unique values
   expect_identical(
-    unique(vaccine_status),
+    sort(unique(vaccine_status), decreasing = TRUE),
     vax_status
   )
 })
```

Comment:
This is probably a mistake I introduced in #13, but it can be fixed by using `expect_setequal()`. No need for `unique()` as well.

```
expect_setequal(vaccine_status, vax_status)
```